### DISCUSSION TITLE
visual selection with 'v' is genius - but could the letters or background color be transparent?

### DISCUSSION THREAD

**Discussion by emilBeBri** [ahrm/sioyek#634] [Ideas]
So I love the visual selection of text with the keyboard. However, it is hard to see the text for the UI elements of two-letters. I can see two ways that would make this easier (ideally: both of them)

1) transparency of the letters-and background 
2) removing the background and having only the two-letters (and then you can make the a standout color like link to discern them from the text of the pdf)

I think that last one would be quite stylish  and would help a lot!

**Comment by ahrm** [comment]
Added `keyboard_select_background_color` and `keyboard_select_text_color` config options in 61527833ca4e985d62e4f343a41f859b10057519. You can now configure the exact background and text colors. The colors take four values (including alpha) so if you want it to be transparent, you can set it like so:
```
keyboard_select_background_color 0 0 0 0
```

**Comment by emilBeBri** [comment]
whoa you are one fast person! It's working perfectly, thank you!
